src/core_modules/capture-core/components/Pages/New/NewPage.actions.js

```diff
@@ -2,12 +2,16 @@ import { actionCreator } from '../../../actions/actions.utils';
 
 export const newPageActionTypes = {
     NEW_PAGE_WITHOUT_ORG_UNIT_SELECTED_VIEW: 'NewPageWithoutOrgUnitSelectedView',
+    NEW_PAGE_WITHOUT_PROGRAM_PARTNER_SELECTED_VIEW: 'NewPageWithoutProgramPartnerSelectedView',
     NEW_PAGE_DEFAULT_VIEW: 'NewPageDefaultView',
 };
 
 
 export const showMessageToSelectOrgUnitOnNewPage = () =>
     actionCreator(newPageActionTypes.NEW_PAGE_WITHOUT_ORG_UNIT_SELECTED_VIEW)();
 
+export const showMessageToSelectProgramPartnerOnNewPage = () =>
+    actionCreator(newPageActionTypes.NEW_PAGE_WITHOUT_PROGRAM_PARTNER_SELECTED_VIEW)();
+
```

Comment:
We use this to display message to the user when they have selected a program which has partners but they havent selected a partner.

Reply:
Yes, "Implementing partner" is the name for the category - I guess that is where the term "Partner" comes from, and we should avoid using this term as @JoakimSM points out. Other programs will have other categories, and the names of these categories is defined in the category metadata. I think for now we change this to "category" as the generic term.

Reply:
So the text when someone hasnt selected a category would be "Choose a category to start reporting".